src/fundus/publishers/us/business_insider.py

```diff
@@ -15,20 +15,13 @@
 
 class BusinessInsiderParser(ParserProxy):
```

Comment:
Some articles do not contain meta keywords, but rather meta news_keywords e.g. https://www.businessinsider.com/guides/health/fitness/best-running-headphones

Reply:
I've updated the topic parsing accordingly.